zcash_client_backend/CHANGELOG.md

```diff
@@ -146,9 +146,8 @@ and this library adheres to Rust's notion of
   - `ChangeValue` is now a struct. In addition to the existing change value, it
     now also provides the output pool to which change should be sent and an
     optional memo to be associated with the change output.
-  - `fixed::SingleOutputChangeStrategy::new` and
-    `zip317::SingleOutputChangeStrategy::new` each now accept an additional
-    `change_memo` argument.
+  - `fixed::SingleOutputChangeStrategy::new` and `zip317::SingleOutputChangeStrategy::new` 
+    each now accept an additional `change_memo` argument
```

Comment:
Ah, this got rearranged because in an intermediate state there was another argument that had been added, and then I just rewrote it without realizing that it should be reverted to the original state.